CHANGES

```diff
@@ -1,6 +1,16 @@
 Version 8.16 (Unreleased)
 -------------------------
 - Added data migration to create UserEmail objects for users whose primary emails did not have them
+- Added distributions to the release system to better support mobile apps
+
+Several schema changes are made in this upgrade:
+
+- A new model: Distribution
+- A new field on Event: dist
+
+Version 8.15
```

Comment:
Because the duplicate was on master and i pulled in at a bad time. Will fix it again.

https://github.com/getsentry/sentry/commit/39a30f53546cd517f78d4956278d77ee1edad0bd